src/lib/toolbar/_toolbar-theme.scss

```diff
@@ -8,6 +8,26 @@
   color: mat-color($palette, default-contrast);
 }
 
+@mixin _mat-toolbar-form-field-overrides {
```

Comment:
@jelbourn @mmalerba I wasn't completely sure whether to put these next to their respective components. I went with having them next to the toolbar, inside a separate mixin, so they don't get too scattered.

Reply:
I think it makes sense to have them here